Objects_And_Functions/js-objects-7.js

```diff
@@ -0,0 +1,25 @@
+// Напишите функцию, которая определяет является ли число простым или нет
+
+function definePrime(number) {
```

Comment:
Works not fully correct. When I passed as number negatives, 0 and 1, I got `The number is prime!`. But it is not prime.